### DISCUSSION TITLE
Unable to understand the different PK profiles between published paper (Britz et al 2020) and my model while replicating

### DISCUSSION THREAD

**Discussion by sash0155** [Open-Systems-Pharmacology/Forum#1767] [Questions & Problems] [answered]
Dear OSP community,

I am trying to use the example studies from published paper of Probenecid from Britz et al 2020 (https://link.springer.com/article/10.1007/s11095-020-02964-z). I have used the same parameters as mentioned in the PK-Sim model of probenecid (same parameters after parameter identification in the model) but I got a different PK profile. I have checked all the tabs/fields to make sure I am not missing anything but still unable to find out the problem. I did my own parameter identification to fit the model but then as expected, my final parameter values are different especially kcat for UGT1A9 and OAT3. I have attached only one study from the paper (I have used other studies as well i.e, Selen 1982_1000mg_p.o._s.d._29 and Vree 1992_1000mg_p.o._s.d._36+37 to develop a model and to see if I was missing any details but the issue was persistent). I would really appreciate if I can get some help to understand the issue in case I am missing anything in the model.

Thanks in advance,
Sajid

The following example is from Emanuelsson 1987 500 mg po study in the probenecid model (1st and 3rd pictures are from original paper and 2nd and 4th are from a replicated model).

P.S. in the 4th picture, I don't know why Resolution didn't show up but I have used the same resolution (20 pts/h).

![Britz 2020 (Emanuelsson 1987 500 mg po) ](https://github.com/Open-Systems-Pharmacology/Forum/assets/146194289/7d0383b5-08f3-4dcd-8042-48af0d1741de)

![My model (Emanuelsson 1987 500 mg po) ](https://github.com/Open-Systems-Pharmacology/Forum/assets/146194289/aa986657-c3c1-4f8d-8010-9ca4acd638a0)

![Britz 2020 parameters (Emanuelsson 1987 500 mg po) ](https://github.com/Open-Systems-Pharmacology/Forum/assets/146194289/4753adae-3c5a-4797-bc3b-e26c55481f2b)

![My model parameters (Emanuelsson 1987 500 mg po) ](https://github.com/Open-Systems-Pharmacology/Forum/assets/146194289/a3445393-7f54-4f88-8fc4-120fe9f7c071)

**Comment by tobiasK2001** [comment]
Dear Sajid,

The model from Britz et al was developed in PK-sim Version 9.
You are likely working in PK-Sim Version 11. Please not that between Version 9 and 11 there has been an substantial change in modeling of Proteins expressions. See details here:

https://github.com/Open-Systems-Pharmacology/Forum/discussions/945

To correct the change for Probenecid you might want to change the Kcat for OAT3 from 1966.57 to eg 1118.93 1/min. With that change you should be able to reproduce the profiles from Britz et al. The kacat for UGT1A9 cis not affected and can be left as it is.

Please let me know if it is working for you.

Best, Tobias

**Reply by sash0155** [reply]
Dear Tobias,

Thanks a lot for your reply and explanation.

I do agree and I had the same thought initially about the different versions of PK-Sim. However, I thought because I am using the Britz model in the same version of 11.3 so at least the deviations should not be there and both models should give the same output. 
As I mentioned, I did my own optimization and the optimal values for OAT3 kcat are 1270.18 1/min and 65 1/min for UGT1A9 kcat and I got the same profile. I also put the value of kcat directly as you suggested and more or less the PK profile was similar to my optimized model. 

This brings me to a couple of questions:
1) We cannot rely on the models (model parameters) which were developed on versions below 11 or 10?
2) Since, there is a huge difference between the parameter values then how can we justify the two different values to someone who wants to design their studies based on these values?

Thank you,
Kind regards,
Sajid

**Reply by StephanSchaller** [reply, marked as answer]
For your questions:
1. if you open a V9 in a V10+ Version, you will see a difference in the simulation only if (one of) the proteins is expressed in interstitial space.
2.The kcats from the model V9 are also fitted. So now you merely re-fit them to adjust for the change of how PK.Sim allocates expressed protein to the tissue subcompartments post V9 (so this change was only from V9 to V10, after that all remains the same). So you merely requalify the model for a new software version. Knowing the "true" kcat depends on if yu really know the accurate expression of the transporter.

**Reply by sash0155** [reply]
Dear Stephan,

Thanks a lot for the clarification.

Kind regards,
Sajid
